tests/test_agents.py

```diff
@@ -54,6 +54,21 @@ def test_add():
     assert l1.direction == Direction.U
     assert l2.direction == Direction.D
 
+def test_RandomAgentProgram() :
+    #create a list of all the actions a vacuum cleaner can perform
+    list = ['Right', 'Left', 'Suck', 'NoOp']
+    # create a program and then an object of the RandomAgentProgram
+    program = RandomAgentProgram(list)
+    
+    agent = Agent(program)
+    # create an object of TrivialVacuumEnvironment
+    environment = TrivialVacuumEnvironment()
+    # add agent to the environment
+    environment.add_thing(agent)
+    # run the environment
+    environment.run()
+    # check final status of the environment
+    assert environment.status == {(1,0): 'Clean' , (0,0): 'Clean'}
```

Comment:
Add a space after the commas in the tuples.

Reply:
The case is same in both the cases t.e, in randomAgentProgram and randomVacuumAgent. Both do choose randomly from the given actions. But as the run method executes the step method a 1000 times, so there is almost 100 percent chance that the assertion will be true always. However, another way of doing this can be declaring actions as an attribute of the function statement and then just executing the step statement once and checking if the action does belong to the given list of actions then the program runs fine. @MrDupin  Comments?

Reply:
OK, sounds good.

There are some conflicts with the `README` now, can you fix them? Thanks.

Reply:
Fixed them @MrDupin